.github/actions/filter-test-configs/action.yml

```diff
@@ -125,7 +125,7 @@ runs:
         TAG: ${{ steps.parse-ref.outputs.tag }}
         EVENT_NAME: ${{ github.event_name }}
         SCHEDULE: ${{ github.event.schedule }}
-        HEAD_BRANCH: ${{ github.event.workflow_run.head_branch }}
+        HEAD_BRANCH: ${{ steps.parse-ref.outputs.branch }}
```

Comment:
Is this because `github.event.workflow_run.head_branch` returns something like `ref/head/main` or something IIRC

Reply:
I think on PRs `github.event.workflow_run.head_branch` is also empty
On PRs the new one seems like it's usually empty (triggered via tag, not branch, like ciflow) or `pull/<pr number>` (triggered via pull request)